src/watchmaker/utils/__init__.py

```diff
@@ -46,6 +46,15 @@ def basename_from_uri(uri):
 @backoff.on_exception(backoff.expo, urllib.error.URLError, max_tries=5)
 def urlopen_retry(uri):
     """Retry urlopen on exception."""
-    # "trust the system's default CA certificates"
-    ctx = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
-    return urllib.request.urlopen(uri, context=ctx)
+    kwargs = {}
```

Comment:
Any idea why this is showing up here? This work was merged in https://github.com/plus3it/watchmaker/pull/633/... maybe a bad commit/amend or something?

Reply:
Comparing the diff of this content and the prior PR and I see zero difference...

Reply:
Hmm... I must have messed up somewhere. I had so many commits that git got all messed up. It listed all the commits in the reverse order and such nonsense.